example-ideal.txt

```diff
@@ -0,0 +1,17 @@
+// Fills an empty room
+// Also demonstrates and tests most language features
+ _      1  |  #      #  |  #      # // This is a comment
+*1* -> * * | *1_ -> * 1 | *1# -> *spam# // yayy
+ *      *  |  *      *  |  *      *
+       
+  *          *   |  *          * 
+ *spam* -> *    *| _spam* -> 22 *
+  _         spam |  #          # 
+// such comment
+ _       22  |  #         #     
+*22* -> *  * |_22* -> spam *  
+  *      *   | *          * 
```

Comment:
Props for staying true to your ideal syntax. This looks pretty much identical to your Empty.bot.